src/main/java/graphql/analysis/RequestWidthInfo.java

```diff
@@ -0,0 +1,65 @@
+package graphql.analysis;
+
+import graphql.PublicApi;
+
+/**
+ * The request width info.
+ */
+@PublicApi
+public class RequestWidthInfo {
```

Comment:
Naming is hard.


Can you justify why you called this Request Width - eg is it because the other instrumentations are request depth ?

I dont hate the name - just trying to understand the reasoning behind it

Reply:
this use case was very similar to query depth instrumentation, so I thought keeping names similar to that would be better.